projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -626,6 +626,25 @@ export interface FeatureTogglesInterface {
    *       in the future together with this feature toggle.
    */
   allPageMetaResolversEnabledInCsr?: boolean;
+
+  /**
+   * When enabled, allows to provide extended formats and media queries for <picture> element if used in MediaComponent.
+   *
+   * For proper work requires `pictureElementFormats`  provided in media config:
+   *  ```ts
+   * provideConfig({
+   *   pictureElementFormats: {
+   *    mediaQueries: {
+   *     'max-width': '767px',
+   *      ...
+   *    },
+   *    width: 50,
+   *    height: 50,
+   *   },
+   * })
+   * ```
+   */
+  useMediaComponentWithConfigurableMediaQueries?: boolean;
```

Comment:
`medium:` After secound thoughts on this topic, I think we should consider changing the name of this feature toggle as it is responsible not only for support of new config, but also controls whether new `MediaComponent` inputs: `width`, `height` and `sizez` are taken into account.
What do you think about `useExtendedMediaComponent`?
The JSDoc should be adjusted too.

Reply:
Done